src/main/java/com/suse/saltstack/netapi/client/impl/HttpClientConnection.java

```diff
@@ -71,7 +71,6 @@ public T getResult() throws SaltStackException {
     /**
      * Perform HTTP request and parse the result into a given result type.
      *
-     * @param method the HTTP method to use
```

Comment:
Correct, @moio, we didn't enable the `JavadocMethod` checkstyle module yet since it requires a bigger patch. Anyways this is fixed with f0ad98c736387151e5937efdab8eb4ebf368b61e.